content/en/events/2022/kcsna/faq.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Frequently Asked Questions (FAQ)"
+type: docs
+weight: 7
+---
+
+- [When is the summit taking place?](#when-is-the-summit-taking-place)
+- [Where is the summit located?](#where-is-the-summit-located)
+- [Will I be able to attend remotely?](#will-i-be-able-to-attend-remotely)
+- [Do I need to wear a mask?](#do-i-need-to-wear-a-mask)
+- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I am not an Org member. Can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
+- [I have an idea for a session.  How do I propose it?](#i-have-an-idea-for-a-session)
+- [My SIG wants to meet at the Summit](#my-sig-wants-to-meet-at-the-summit)
+- [Will there be a Doc Sprint?](#will-there-be-a-doc-sprint)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [Can I bring a guest to the Contributor Social](#can-i-bring-a-guest-to-the-social)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+October 24th, the Monday of [KubeCon North America].
+
+[KubeCon Europe]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Huntington Place Convention Center</a>,
+See the [Location & Venue] section for more information.
+
+[Location & Venue]: /events/2022/kcsna/location/
+
+
+### Will I be able to attend remotely?
+
+No, based on our experience with Kubecon NA 2021, we do not expect to be able
+to support remote access for this event.
+
+
+### Do I need to wear a mask?
+
+All attendees are required to wear masks except when eating or drinking.  Eating and drinking should be kept to designated areas.
```

Comment:
nit 😉: contemporary writing styles suggest a single space after a period (see APA, AP, MLA, Chicago, Microsoft 😄 )
```suggestion
All attendees are required to wear masks except when eating or drinking. Eating and drinking should be kept to designated areas.
```

Reply:
lol that was not intentional, fixed it as well as a few others.